src/object/obj_ec.h

```diff
@@ -443,23 +489,24 @@
 						obj_ec_stripe_rec_nr(oca);
 	uint64_t			 cell_rec_nr =
 						obj_ec_cell_rec_nr(oca);
-	uint32_t			 i, j, stripe_nr;
+	uint32_t			 i, j;
 
 	if (lists == NULL)
 		return;
 	for (i = 0; i < nr; i++) {
 		list = &lists[i];
 		for (j = 0; j < list->re_nr; j++) {
-			recx = &list->re_items[j].re_recx;
+			uint64_t end;
+
+ 			recx = &list->re_items[j].re_recx;
+ 			D_ASSERT((recx->rx_idx & PARITY_INDICATOR) != 0);
+ 			recx->rx_idx &= ~PARITY_INDICATOR;
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs